From 2c5d72974e08333d09e77c0027c065e512738973 Mon Sep 17 00:00:00 2001
Message-Id: <2c5d72974e08333d09e77c0027c065e512738973.1354540818.git.minovotn@redhat.com>
From: Eduardo Habkost <ehabkost@redhat.com>
Date: Wed, 14 Nov 2012 18:46:52 +0100
Subject: [PATCH 1/3] Recognize PCID feature

RH-Author: Eduardo Habkost <ehabkost@redhat.com>
Message-id: <1352918814-16583-2-git-send-email-ehabkost@redhat.com>
Patchwork-id: 44219
O-Subject: [RHEL6.4 qemu-kvm PATCH 1/3] Recognize PCID feature
Bugzilla: 869214
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>
RH-Acked-by: Igor Mammedov <imammedo@redhat.com>
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>

From: "Mao, Junjie" <junjie.mao@intel.com>

Bugzilla: 869214

This patch makes Qemu recognize the PCID feature specified from configuration or command line options.

Signed-off-by: Junjie Mao <junjie.mao@intel.com>
Signed-off-by: Stefan Hajnoczi <stefanha@linux.vnet.ibm.com>
(cherry picked from commit 434acb817b8ae747f31e91ec152f9f47ac514433)

Conflicts:
	target-i386/cpu.c

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 target-i386/cpuid.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 target-i386/cpuid.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target-i386/cpuid.c b/target-i386/cpuid.c
index 3de8d50..4312112 100644
--- a/target-i386/cpuid.c
+++ b/target-i386/cpuid.c
@@ -48,7 +48,7 @@ static const char *ext_feature_name[] = {
     "ds_cpl", "vmx", "smx", "est",
     "tm2", "ssse3", "cid", NULL,
     "fma", "cx16", "xtpr", "pdcm",
-    NULL, NULL, "dca", "sse4.1|sse4_1",
+    NULL, "pcid", "dca", "sse4.1|sse4_1",
     "sse4.2|sse4_2", "x2apic", "movbe", "popcnt",
     "tsc-deadline", "aes", "xsave", "osxsave",
     "avx", "f16c", "rdrand", "hypervisor",
-- 
1.7.11.7

